cumulusci/tasks/bulkdata/mapping_parser.py

```diff
@@ -124,3 +293,36 @@ def validate_mapping(cls, values):
 def parse_from_yaml(source: Union[str, Path, IO]) -> Dict:
     "Parse from a path, url, path-like or file-like"
     return MappingSteps.parse_from_yaml(source)
+
+
+def validate_mapping(
```

Comment:
"validate" for me implies that it will check status but not mutate. Can we give it a name that makes it clearer that mapping may be changed by running this?

Reply:
Renaming to `validate_and_inject_mapping()`. This method is admittedly a bit of a chimera. It's efficient to do this in a single operation, though, because the describe logic is in common.